components/todo-item.js

```diff
@@ -1,11 +1,63 @@
-import React from "react";
+import React, { useState } from 'react';
+
 import styled from "styled-components";
 
-export default function TodoItem() {
-  return <Wrapper>Todo</Wrapper>;
+export default function TodoItem({ id, content, date, todoRemove }) {
+  return (
+    <Wrapper>
+      <TodoItemBox>
+        <Todo>{content}</Todo>
+        <Day>{date}
+        <ClearBtn onClick={(() => {todoRemove(id)})}>완료</ClearBtn>
+        </Day>
+      </TodoItemBox>
+    </Wrapper>
+    );
 }
 
 const Wrapper = styled.div`
-  font-size: 18px;
-  border: solid 1px;
+  display: flex;
+`;
+
+const TodoItemBox = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin-bottom: 3rem;
+  padding: 1rem 2rem;
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+  border-image: initial;
+`;
+
+const Todo= styled.p`
+  font-size: 1.5rem;
+  display: flex;
+  flex-direction: column;
+  padding: 0px;
+  margin: 0px;
+`;
+
+const Day= styled.p`
+  font-size: 1.5rem;
+  display: flex;
+  flex-direction: column;
+  padding: 0px;
+  margin: 0px;
+`;
+
+const ClearBtn = styled.button`
+  color: white;
+  background-color: rgb(97, 97, 97);
+  font-size: 1.5rem;
+  outline: none;
+  border-width: initial;
+  border-style: none;
+  border-color: initial;
+  border-image: initial;
+  padding: 0.5rem 1rem;
+  border-radius: 0.3rem;
 `;
```

Comment:
```suggestion
const DeleteButton = styled.button`
  color: white;
  background-color: rgb(97, 97, 97);
  font-size: 1.5rem;
  outline: none;
  padding: 0.5rem 1rem;
  border: none;
  border-radius: 0.3rem;
  margin-left: auto;
`;
```
1. 네이밍은 ```components/todo-input```의 ```SubmitButton```과 같은 이유로 ```DeleteButton```으로 바꿔주시면 좋아요~
2. DeleteButton을 오른쪽 정렬하고 싶으시다면 ```margin-left: auto```를 주시면 됩니다. [참고](https://developer.mozilla.org/ko/docs/Web/CSS/margin-left#Values)